src/wp-includes/sodium_compat/lib/php72compat.php

```diff
@@ -1360,6 +1360,8 @@ function sodium_memcmp($string1, $string2)
      * @return void
      * @throws SodiumException
      * @throws TypeError
+     *
+     * @psalm-suppress ReferenceConstraintViolation
```

Comment:
While this isn't strictly necessary, it seemed like this change was worth adding to minimize differences with the original source.

Reply:
👍🏻 I agree with this reasoning.